interfaces/builtin/avahi_control.go

```diff
@@ -119,9 +118,10 @@ func (iface *avahiControlInterface) StaticInfo() interfaces.StaticInfo {
 func (iface *avahiControlInterface) AppArmorConnectedPlug(spec *apparmor.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
 	old := "###SLOT_SECURITY_TAGS###"
 	var new string
-	if release.OnClassic {
-		// If we're running on classic Avahi will be part
-		// of the OS snap and will run unconfined.
+	// If we're running on classic Avahi can be installed as snap or be part
+	// of the OS and it will run unconfined.
```

Comment:
Does this comment need to be expanded? Something like:
```
	// If we're running on classic Avahi can be installed as snap or be part
	// of the OS and it will run unconfined *or* it can be installed as a snap
        // with the normal confined snap label
```

Reply:
updated